src/main.cpp

```diff
@@ -17,7 +17,16 @@ int main(int argc, char *argv[]) {
 
     ArgsParser parser;
     VerificationOptions options = parser.parse(argc, argv);
-    unfoldtacpn::ColoredPetriNetBuilder builder;
+
+    
+    std::stringstream* output_stream = nullptr;
+    if (options.getPrintBindings()) {
+        output_stream = new std::stringstream{};
+    }
+
+    unfoldtacpn::ColoredPetriNetBuilder builder(output_stream);
```

Comment:
if you dont user pointers, send 
options.getPrintBindings() ? &output_stream : nullptr